### ISSUE TITLE
Remove `desc` from control logic

### ISSUE THREAD

**Issue by karikarshivani** [mitre/stig-microsoft-windows-server-2016-v1r4-baseline#59]
@damif5 found `desc` being used before a describe block with skip statements - which would override the original description in certain controls. Here is an example snippet:
```rb
  impact 0.0
  desc 'This system is not a domain controller, therefore this control is not applicable as it only applies to domain controllers'
  describe 'This system is not a domain controller, therefore this control is not applicable as it only applies to domain controllers' do
    skip 'This system is not a domain controller, therefore this control is not applicable as it only applies to domain controllers'
  end
```

The test result will still display the describe line and the skip line so `desc` is unnecessary and should be removed.

Rough list of controls:
- [x] V-73219
- [x] V-73221
- [x] V-73227
- [x] V-73233
- [x] V-73247
- [x] V-73257
- [x] V-73303
- [x] V-73359
- [x] V-73361
- [x] V-73363
- [x] V-73365
- [x] V-73367
- [x] V-73369
- [x] V-73371
- [x] V-73375
- [x] V-73377
- [x] V-73379
- [x] V-73381
- [x] V-73383
- [x] V-73385
- [x] V-73387
- [x] V-73389
- [x] V-73391
- [x] V-73393
- [x] V-73395
- [x] V-73397
- [x] V-73399
- [x] V-73435
- [x] V-73437
- [x] V-73439
- [x] V-73441
- [x] V-73513
- [x] V-73515
- [x] V-73517
- [x] V-73607
- [x] V-73609
- [x] V-73611
- [x] V-73613
- [x] V-73615
- [x] V-73617
- [x] V-73629
- [x] V-73631
- [x] V-73731
- [x] V-73733
- [x] V-73737
- [x] V-73741
- [x] V-73757
- [x] V-73759
- [x] V-73761
- [x] V-73763
- [x] V-73765
- [x] V-73767
- [x] V-73769
- [x] V-73771
- [x] V-73773
- [x] V-73775
- [x] V-73777
- [x] V-73779
- [x] V-78123
- [x] V-78125

**Comment by abailey1021** [comment]
Desc lines removed from listed controls as requested. 6/13/23

**Comment by karikarshivani** [comment]
Addressed by #62
